content/quickstart.md

```diff
@@ -73,22 +73,29 @@ $ export GITHUB_CLIENT_ID=...
 $ export GITHUB_CLIENT_SECRET=...
 ```
 
-Install the OpenShift template (syndesis-dev.yml or syndesis.yml as discussed [above]({{< relref "#template-selection" >}})):
+Install the OpenShift template (syndesis-restricted-dev.yml or syndesis-restricted.yml as discussed [above]({{< relref "#template-selection" >}})):
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/syndesis-dev.yml
+$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/syndesis-restricted-dev.yml
```

Comment:
I wonder whether we should always point to `master` or to a released version of the templates (where we also have pinned the image versions) ?